app/code/Magento/Bundle/Model/Product/CopyConstructor/Bundle.php

```diff
@@ -6,28 +6,38 @@
 namespace Magento\Bundle\Model\Product\CopyConstructor;
 
 use Magento\Catalog\Model\Product;
+use Magento\Catalog\Model\Product\CopyConstructorInterface;
 use Magento\Catalog\Model\Product\Type;
 
-class Bundle implements \Magento\Catalog\Model\Product\CopyConstructorInterface
+class Bundle implements CopyConstructorInterface
 {
     /**
      * Duplicating bundle options and selections
-     *
      * @param Product $product
      * @param Product $duplicate
      * @return void
      */
-    public function build(Product $product, Product $duplicate)
+    public function build(Product $product, Product $duplicate): void
```

Comment:
Please remove the strict :) "void" return type declaration as it can result in a warning on instances where the class is extended